source/common/common/to_lower_table.cc

```diff
@@ -0,0 +1,16 @@
+#include "common/common/to_lower_table.h"
+
+ToLowerTable::ToLowerTable() {
```

Comment:
The way I've done this before is a constexpr function that builds and returns the lookup array, then that function is used to initialize a const static std::array.  This allows you to have a big block of const data with arbitrarily complex initialization done at compile time, as long as all data used by the initialization is known at compile time.